plugins/modules/discovery_intent.py

```diff
@@ -200,6 +200,101 @@
                     - Requires valid SSH credentials to work.
                     - Avoid standard ports like 22, 80, and 8080.
                 type: str
+      global_credentials:
+        description:
+            - Credentials that are already created by the user under Device Credentials in Cisco Catalyst Center.
```

Comment:
`Set of various credential types, including CLI, SNMP, HTTP, and NETCONF, that a user has pre-configured in the Device Credentials section of the Cisco Catalyst Center.
`

Reply:
Added